.github/workflows/cron.yml

```diff
@@ -128,3 +128,26 @@ jobs:
       with:
         fail_ci_if_error: false
         file: ./coverage.xml
+
+  cron-tutorial-notebooks:
+    if: github.repository == 'Project-MONAI/MONAI'
+    container:
+      image: nvcr.io/nvidia/pytorch:21.02-py3  # testing with the latest pytorch base image
+      options: "--gpus all"
+    runs-on: [self-hosted, linux, x64, common]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        which python
+        python -m pip install --upgrade pip wheel
+        python -m pip install ${{ matrix.pytorch }}
+        python -m pip install -r requirements-dev.txt
+    - name: Checkout tutorials and install their requirements
+      run: |
+        git clone https://github.com/Project-MONAI/tutorials.git
+        cd tutorials
+        python -m pip install -r requirements.txt
+    - name: Run tutorial notebooks
+      run: |
+        sh ./runtests.sh
```

Comment:
All our scripts are bash scripts so this should be `bash` in case `sh` isn't an alias to it. Also is the script called `runtests.sh` in tutorials? There's a script file there called `runner.sh` which refers to itself as `runtests.sh`.

Reply:
Thanks. Updated to using `$(pwd)/runner.sh` (`$(pwd)` is used for other bash scripts). Also made a PR for `runner.sh` referring to itself as `runtests.sh`.